wforce/wforce-replication.cc

```diff
@@ -97,6 +104,11 @@ void replicateOperation(const ReplicationOperation& rep_op)
   encryptMsg(msg, packet);
 
   for(auto& s : *siblings) {
+    if (s->d_has_key) {
+      if (s->d_key != g_key) {
+        encryptMsgWithKey(msg, packet, s->d_key, s->d_nonce, s->mutx);
```

Comment:
I must be missing something but I don't get why `msg` is added multiple times to `packet`

Reply:
Yes, this is a bug. It should use the original packet if there is no unique key for the sibling, otherwise it should encrypt msg into a different packet variable.